src/core/webidl.js

```diff
@@ -49,9 +51,22 @@ const templates = {
       case "object":
         type = "interface";
         cite = "WebIDL";
+        break;
+      default: {
+        const isWorkerType = unescaped.includes("Worker");
```

Comment:
We also have worklets: https://webaudio.github.io/web-audio-api/#audioworkletglobalscope

Reply:
Let’s add these in the next round.